skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/SkribentenDatabaseService.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.pensjon.brev.skribenten.services
+
+import com.typesafe.config.Config
+import com.zaxxer.hikari.HikariConfig
+import com.zaxxer.hikari.HikariDataSource
+import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.eq
+import org.jetbrains.exposed.sql.transactions.transaction
+
+
+class SkribentenDatabaseService {
+    fun getFavourites(userId: String): List<String> =
+        transaction {
+            Favourites.select { Favourites.userId eq userId }.map { row -> row[Favourites.letterCode] }
+        }
+
+
+    fun addFavourite(userID: String, letterId: String) {
+        transaction {
+            Favourites.insert {
+                it[userId] = userID
+                it[letterCode] = letterId
+            }
+        }
+    }
+
+    fun removeFavourite(userID: String, letterId: String) {
+        transaction {
+            Favourites.deleteWhere {
+                userId eq userID
+                letterCode eq letterId
+            }
+        }
+    }
+}
+
+
+@Suppress("unused")
+object Favourites : Table() {
```

Comment:
Tolkes non-nullable typen her for skjemaet? Slik at alle kollonene er påkrevd. Er 50 litt kort for userId?

Reply:
Etter jeg har endret til NAV-id er det ikke for kort. Det valideres både i controlleren og via exposed.  Sjekket databasen og den påkrever not null ja.